app/components/homepage-components/FeaturedProjectCard.tsx

```diff
@@ -9,7 +9,7 @@ const FeaturedProjectCard = ({
 }: FeaturedProjectCardProps) => {
   return (
     <div
-      className="flex w-full max-w-[30rem] flex-col overflow-hidden rounded-xl px-6 py-12 lg:max-w-7xl lg:flex-row lg:pl-[7rem] lg:pt-[6rem]"
+      className="flex w-full max-w-xl flex-col overflow-hidden rounded-xl px-6 py-12 lg:max-w-7xl lg:flex-row lg:pl-[5rem] lg:pt-[6rem]"
       style={{ backgroundColor: project.backgroundColour }}
     >
```

Comment:
Nice catch with max-w-xl! 

Also, I noticed you've utilized the style attribute to set the background, and I'm curious about your choice for this approach? Consider defining the color in your Tailwind config and using it with Tailwind classNames, as you've done previously in your app. It's essential to be consistent across your application as it can significantly enhance readability and maintainability. Please let me know if there is another reason behind it.

Reply:
@cdsalty Hi Christopher. I used the style attribute here because I was having issues passing the background colour dynamically into the classname because it was something like bg-[${backgroundColour}] within the class name. Eslint was making it behave in some strange ways and wasn't able to read this properly in the classname. I had to do this on the last project as well in a couple of places. The only way I could get it to work all the time so far was to use the style attribute. I don't like having to do this because it defeats the point of tailwind but I wasn't sure what else to do.